interface/super/edit_globals.php

```diff
@@ -126,9 +126,30 @@ function checkBackgroundServices(){
       foreach ($grparr as $fldid => $fldarr) {
         if (in_array($fldid, $USER_SPECIFIC_GLOBALS)) {
           list($fldname, $fldtype, $flddef, $flddesc, $fldlist) = $fldarr;
+          //check and validate input from client side with globals.
+          if(is_array($grparr[$fldid][1])) {
+          $search = $grparr[$fldid][1];
+          $boolean = array_key_exists(trim(strip_escape_custom($_POST["form_$i"])),$search);
+          }
+          elseif ($grparr[$fldid][1] == bool) {
+          $_POST["form_$i"] == 0;
+          $boolean = true;
+          }
+          elseif ($fldid == "css_header") {
+            //styles array created since the globals does not have styles array.
+            $styles_array = array("style_prism.css", "style_light.css", "style_purple.css", "style_tan.css", "style_tan_no_icons.css");
```

Comment:
Can't have anyone inserting their own CSS here now... :)  TODO, figure out where the enumeration should come from.  Probably against existing database values, vs hard-coding.